pkg/apis/core/validation/utils.go

```diff
@@ -134,3 +134,17 @@ func equal(new, old []string) bool {
 	}
 	return true
 }
+
+func isIgnoreTaintsUnique(ignoredTaints []string, fldPath *field.Path) field.ErrorList {
```

Comment:
To be consistent with all similar helper funcs (such as `validateKubeletConfigEviction`, `validateTaints`, etc). The `is` prefix rather suggests that the func returns bool - in our case we perform validation and return `field.ErrorList`.

```suggestion
func validateClusterAutoscalerIgnoreTaints(ignoredTaints []string, fldPath *field.Path) field.ErrorList {
```